lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context.cpp

```diff
@@ -182,6 +263,79 @@ amf_context_t* amf_context_get(const amf_ue_ngap_id_t ue_id) {
+ **                                                                        **
+ ** Name:    amf_ue_context_exists_imsi()                                  **
+ **                                                                        **
+ ** Description: Checks if UE context exists for IMSI or not               **
+ **                                                                        **
+ **                                                                        **
+ ***************************************************************************/
+struct ue_m5gmm_context_s* amf_ue_context_exists_imsi(
+    amf_ue_context_t* const amf_ue_context_p, imsi64_t imsi64) {
+  magma::map_rc_t m_rc = magma::MAP_OK;
+  uint64_t amf_ue_ngap_id64 = 0;
+
+  m_rc = amf_ue_context_p->imsi_amf_ue_id_htbl.get(imsi64, &amf_ue_ngap_id64);
+  if (m_rc == magma::MAP_OK) {
+    return amf_ue_context_exists_amf_ue_ngap_id(
+        (amf_ue_ngap_id_t)amf_ue_ngap_id64);
+  } else {
+    OAILOG_WARNING_UE(LOG_AMF_APP, imsi64,
+                      " No IMSI hashtable for this IMSI\n");
+  }
+  return NULL;
+}
+
+/****************************************************************************
+ **                                                                        **
+ ** Name:    amf_get_ue_context_from_imsi()                                **
+ **                                                                        **
+ ** Description: Fettch the UE context from IMSI                           **
+ **                                                                        **
+ **                                                                        **
+ ***************************************************************************/
+ue_m5gmm_context_s* amf_get_ue_context_from_imsi(char* imsi) {
+  amf_context_t* amf_context_p = nullptr;
+  imsi64_t imsi64 = INVALID_IMSI64;
+
+  IMSI_STRING_TO_IMSI64((char*)imsi, &imsi64);
+
+  amf_app_desc_t* amf_app_desc_p = get_amf_nas_state(false);
+
+  return (amf_ue_context_exists_imsi(&amf_app_desc_p->amf_ue_contexts, imsi64));
+}
+
+/****************************************************************************
+ **                                                                        **
+ ** Name:    amf_ue_context_exists_guti()                                  **
+ **                                                                        **
+ ** Description: Checks if UE context exists for GUTI or not               **
+ **                                                                        **
+ **                                                                        **
+ ***************************************************************************/
+ue_m5gmm_context_s* amf_ue_context_exists_guti(
+    amf_ue_context_t* const amf_ue_context_p, const guti_m5_t* const guti_p) {
+  magma::map_rc_t m_rc = magma::MAP_OK;
+  uint64_t amf_ue_ngap_id64 = 0;
+  ue_m5gmm_context_t* ue_context = NULL;
```

Comment:
For pointer, it's preferred as ue_context_p

Reply:
Comment addressed.

Reply:
Comment addressed.